tests/fixture/e2e/metrics_link.go

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package e2e
+
+import (
+	"strconv"
+	"time"
+
+	"github.com/ava-labs/avalanchego/tests/fixture/tmpnet"
+
+	ginkgo "github.com/onsi/ginkgo/v2"
+)
+
+// The ginkgo event handlers defined in this file will be automatically
+// applied to all ginkgo suites importing this package.
+
+// Whether a spec-scoped metrics link should be emitted after the current
+// spec finishes executing.
+var EmitMetricsLink bool
```

Comment:
Is there a reason this is defined as a global rather than being on the `TestEnvironment`?

Reply:
I guess the global `TestEnvironment` isn't actually specialized for individual tests.

Reply:
I always feel strongly - but I don't think we need to do it in this PR haha